src/metrics/metrics.go

```diff
@@ -50,8 +61,21 @@ func PopulateInstanceMetrics(instanceEntity *integration.Entity, connection *con
 	}
 
 	populateWaitTimeMetrics(instanceEntity, connection)
-	if arguments.CustomMetricsQuery != "" {
-		populateCustomMetrics(instanceEntity, connection, arguments.CustomMetricsQuery)
+	if len(arguments.CustomMetricsQuery) > 0 {
+		populateCustomMetrics(instanceEntity, connection, []CustomQuery{{Query: arguments.CustomMetricsQuery}})
+	} else if len(arguments.CustomMetricsConfig) > 0 {
+		// load YAML config file
+		b, err := ioutil.ReadFile(arguments.CustomMetricsConfig)
+		if err != nil {
+			log.Fatal(err)
+		}
+		// parse
+		var c struct {Queries []CustomQuery}
```

Comment:
This can be simplified to just `var queries []CustomQuery`

Reply:
The YAML has
```
queries:
  - query: SELECT * FROM table
```
So it needs one more level of hierarchy.